### PR TITLE
translate zh-cn: Minor changes to README file

### PR DESCRIPTION
**Pull request by max-sun-hongyi**
I have reviewed and changed a few minor wording preferences.

Changed the translation for "knowledge checks" from "知识检查" to "知识测验"

### CODE REVIEW

**Comment by max-sun-hongyi** [comment]
@Mireille-T can you please review this PR when possible.

**Review by Mireille-T — CHANGES_REQUESTED** [review]
Hello @max-sun-hongyi , thanks a lot for your translations! Regarding the change of 知识检查 to 知识测验, in an earlier PR @NeroBlackstone changed it to 课前测验 and 课后测验, which I think are better translations; I'll work on changing those in the other translated documents when possible, but for now may I trouble you to make the changes in this file before the PR is merged?

Other than that, everything looks fine to me on the rich diff, thank you once again! :smile:

File: translations/README.zh-cn.md [hunk 1]
```diff
@@ -10,23 +10,22 @@
 
 # 物联网（IoT for Beginners） – 课程
 
-Microsoft 的 Azure Cloud 大使很高兴提供关于 IoT 基础一个12个星期、24课的课程。每一课包括课前和课后的知识检查、为了完成课的指示、解答、作业等。我们通过项目的教学让你一边创建，一边学习——一种有效率保留新技能的方式。
+Microsoft 的 Azure Cloud 大使很高兴提供关于 IoT 基础一个12个星期、24课的课程。每一课包括课前和课后的知识测验、为了完成课的指示、解答、作业等。我们通过项目式的教学让你一边创建，一边学习，这是一种有效率巩固新技能的方式。
```
Comment (max-sun-hongyi) [inline translations/README.zh-cn.md:13]:
```suggestion
Microsoft 的 Azure Cloud 大使很高兴提供关于 IoT 基础一个12个星期、24课的课程。每一课包括课前测验和课后测验、为了完成课的指示、解答、作业等。我们通过项目式的教学让你一边创建，一边学习，这是一种有效率巩固新技能的方式。
```

**Review by max-sun-hongyi — COMMENTED** [review]
Changed to 课前测验 and 课后测验 where appropriate.
Resolved conflict with the main branch

**Review by Mireille-T — APPROVED** [review]
Thank you @max-sun-hongyi !

@jimbobbennett This PR can be merged. :smile:

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**